scripts/ci/debian/compat

```diff
@@ -0,0 +1 @@
+9
```

Comment:
I would lower this to 7 - we aren't using (or supporting) any features from dh 9. Or indeed 8.